lta/rest_server.py

```diff
@@ -903,6 +948,14 @@ def start(debug: bool = False) -> RestServer:
     motor_client = MotorClient(lta_mongodb_url)
     args['db'] = motor_client[mongo_db]
 
+    # configure access to MongoDB for health checks
+    mongodb_health_url = f"mongodb://{mongo_host}:{mongo_port}/admin"
+    if mongo_user and mongo_pass:
+        mongodb_health_url = f"mongodb://{mongo_user}:{mongo_pass}@{mongo_host}:{mongo_port}/admin"
```

Comment:
The one issue I can foresee already is which db the user/pass combo is registered in.  Right now, the user only exists in the specific `$LTA_MONGODB_DATABASE_NAME`, not in `admin`.

One solution is to only use the admin db, and when connecting do things like this:

```python
    logging.info(f'DB: {config["DB_URL"]}')
    db_url, db_name = config['DB_URL'].rsplit('/', 1)
    db = motor.motor_asyncio.AsyncIOMotorClient(db_url)
    logging.info(f'DB name: {db_name}')
    kwargs['db'] = db[db_name]
```